jobs/pr_gate/docker/monorail/config.json

```diff
@@ -0,0 +1,64 @@
+{
+    "CIDRNet": "172.31.128.0/22",
+    "amqp": "amqp://localhost",
+    "apiServerAddress": "172.31.128.250",
```

Comment:
Why 172.31.128.250?
I recalled every Jenkins slave use this IP as a local apach2 file server  for isofarm